### DISCUSSION TITLE
Downloads down

### DISCUSSION THREAD

**Discussion by poligit** [primefaces/community#2725] [PrimeFaces] [answered]
https://www.primefaces.org/downloads
404 - Page not found

**Comment by melloware** [comment]
cc @cagataycivici

**Reply by melloware** [reply]
@poligit were you looking for something specific?

**Reply by poligit** [reply]
Yes, I was looking for PrimeFaces jar library.

**Reply by melloware** [reply, marked as answer]
you can also download them from here: https://github.com/primefaces/primefaces?tab=readme-ov-file#versions

**Reply by poligit** [reply]
Thanks!
